nodewatcher/core/generator/cgm/models.py

```diff
@@ -397,7 +397,8 @@ def get_index(self):
 
         return list(vifs).index(self.pk)
 
-registration.point('node.config').register_choice('core.interfaces#wifi_mode', registration.Choice('mesh', _("Mesh")))
+registration.point('node.config').register_choice('core.interfaces#wifi_mode', registration.Choice('mesh', _("Mesh (ad-hoc)")))
```

Comment:
Can we rename this to `mesh-ah` and the other one to `mesh-11s` or something? But then we need a migration. So maybe not. :-)